pkg/controllers/provisioning/scheduling/suite_test.go

```diff
@@ -3323,6 +3323,7 @@ var _ = Describe("Binpacking", func() {
 					v1.ResourceCPU:    resource.MustParse("1"),
 				},
 			},
+				InitImage: "pause",
```

Comment:
The reason I used alpine is because pause never exits, and prevents the pod from starting. Not an issue in functional tests, but I got thrown by it.